spec/puppet-server-lib/puppet/jvm/settings_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+require 'puppet/server/settings'
+
+describe 'Puppet::Server::Settings' do
+
+  before :each do
+    @old_settings = Puppet.settings
+    Puppet.replace_settings_object(Puppet::Settings.new)
+    Puppet.initialize_default_settings!(Puppet.settings)
+
+
+    @new_max_errors = (Puppet[:max_errors] + 10)
```

Comment:
Why parentheses here?

Reply:
Oh, that's leftover from when I had that interpolated elsewhere. Whoops.